### ISSUE TITLE
Enhancement in documentation: Chart description documentation is outdated

### ISSUE THREAD

**Issue by SeppPenner** [PhilJay/MPAndroidChart#3987]
Your documentation for the **description** methods for a chart is outdated.

https://github.com/PhilJay/MPAndroidChart/wiki/General-Chart-Settings-&-Styling:

- setDescription(String desc): Set a description text that appears in the bottom right corner of the chart.
- setDescriptionColor(int color): Sets the color of the description text.
- setDescriptionPosition(float x, float y): Sets a custom position for the description text in pixels on the screen.

However, the Chart.java class only contains a method to set a Description object as a total object:

```java
public void setDescription(Description desc) {
	this.mDescription = desc;
}
```

So, to set e.g. the titel of a desxcription to the chart, you need to do the following (assuming Kotlin, but is quite equal to Java):

```kotlin
private fun something() {
	myChart.description = getDescription()
}

private fun getDescription(): Description{
	val description = Description()
	description.text = getString(R.string.consumptionPer100km)
	//Here e.g. the setPosition(float x, float y) or setTextAlign(Paint.Align align) methods can be applied,
	//too
	return description
}
```

I think, the documentation should be updated as this is really confusing otherwise.

**Comment by almic** [comment]
Yes, since the library has been... er... ignored... for the most part... for around a year now, the documentation in the Wikis is very limited and sometimes wrong.

Step one will be updating the javadoc, and then step two is updating the Wiki to be more extensive.

As far as I know, I'm the only one left who is working on this project. This is definitely on my to-do list.
